mysql-test/suite/innodb/t/innochecksum_undo_page.test

```diff
@@ -0,0 +1,47 @@
+--source include/have_innodb.inc
+--source include/have_sequence.inc
+--source include/not_embedded.inc
+let MYSQLD_DATADIR= `SELECT @@datadir`;
+
+SET GLOBAL INNODB_FILE_PER_TABLE= 0;
+CREATE TABLE t1(f1 INT NOT NULL)ENGINE=InnoDB;
+--disable_query_log
+let $c = 16;
+while ($c)
+{
+  connect (con$c,localhost,root,,);
+  dec $c;
+}
+
+let $c = 16;
+while ($c)
+{
+  connection con$c;
+  START TRANSACTION;
+  eval INSERT INTO t1 select seq from seq_1_to_128;
+  dec $c;
+}
+
+--disable_query_log
```

Comment:
The two first `while` loops can be combined: `connect` implies a change of `connection`.

Do we really need this many connections and rows? If yes, then I think that we need `big_test.inc` to avoid overloading our CI system.